resources/migrations/000_migrations.yaml

```diff
@@ -11636,6 +11636,240 @@ databaseChangeLog:
             tableName: persisted_info
             columnName: creator_id
             columnDataType: int
+  - changeSet:
+      id: v44.00-012
+      author: snoe
+      comment: Added 0.44.0 - writeback
+      changes:
+        - createTable:
+            tableName: action
+            remarks: An action is something you can do, such as run a readwrite query
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  remarks: The timestamp of when the timeline was created
+                  name: created_at
+                  type: ${timestamp_type}
+                  defaultValueComputed: current_timestamp
+                  constraints:
+                    nullable: false
+              - column:
+                  remarks: The timestamp of when the timeline was updated
+                  name: updated_at
+                  type: ${timestamp_type}
+                  defaultValueComputed: current_timestamp
+                  constraints:
+                    nullable: false
+              - column:
+                  remarks: Type of action
+                  name: type
+                  type: ${text.type}
+                  constraints:
+                    nullable: false
+  - changeSet:
+      id: v44.00-013
+      author: snoe
+      comment: Added 0.44.0 - writeback
+      changes:
+        - createTable:
+            tableName: query_action
+            remarks: A readwrite query type of action
+            columns:
+              - column:
+                  name: action_id
+                  type: int
+                  remarks: The related action
+                  constraints:
+                    nullable: false
+                    referencedTableName: action
+                    referencedColumnNames: id
+                    foreignKeyName: fk_query_action_ref_action_id
+                    deleteCascade: true
+              - column:
+                  name: card_id
+                  type: int
+                  remarks: The related card
+                  constraints:
+                    nullable: false
+                    referencedTableName: report_card
+                    referencedColumnNames: id
+                    foreignKeyName: fk_query_action_ref_card_id
+                    deleteCascade: true
```

Comment:
Not sure if we're going to want an index on this column or not. At some point in the future maybe we want to be able to say "where is this readwrite Saved Question used as an Action"? And when deleting a Saved Question it would be nice if we could delete the associated `query_action` rows without doing a full table scan on `query_action`. Might be a good idea to create an index of `action_id` as well to prevent full table scans there.

It's probably not a big deal in the short term for prototype purposes so I'm ok with opening a GitHub issue to revisit the indexes here and adding it to #22542 so we don't forget to do it. We might end up throwing out some of these new tables anyway so maybe it's too soon to try to make everything perfect

Reply:
I'd rather get this in sooner to unblock other work and worry about adding more indexes later